rollup.config.js

```diff
@@ -1,10 +1,4 @@
-import buble from 'rollup-plugin-buble'
-import commonjs from 'rollup-plugin-commonjs'
-import resolve from 'rollup-plugin-node-resolve'
-import uglify from 'rollup-plugin-uglify'
-import serve from 'rollup-plugin-serve'
-import livereload from 'rollup-plugin-livereload'
-import postcss from 'rollup-plugin-postcss'
+const $ = require('rollup-load-plugins')()
```

Comment:
I like this idea but how come you are using `require` here and not `import`?

Reply:
hi @lukejacksonn, sorry I somehow missed this notification. I believe we should convert all the imports in rollup config to require since current node LTS does not support import just yet. Will have another commit to convert all of them if we are on same page.

Reply:
humm.. you are probably right. On the topic of loading plugins.. I made this https://github.com/lukejacksonn/preroll so that I could import all of this behaviour standardly across projects. What do you think about this abstraction (supposed to adding all these rollup deps to each project). It feels like _prettier_ but for builds.. too far?

Reply:
The abstraction is awesome, but breaks if we want it to be configurable. If you make it configurable, you are essentially creating a new DSL (or rather abstract DSL) for rollup configuration. 
Personally, I would not make it a part of generic startup to keep things simple for newcomers, but use it for rest of my projects. 
Also, recently, I am using a fantastic tool called [mrm](https://github.com/sapegin/mrm) by @sapegin which helps me keeping the configurations in sync for multiple projects. YMMV, but it's worth a try.